autofs-5.1.2 - update and add README for old autofs schema

From: Ian Kent <raven@themaw.net>

The distribution file samples/autofs.schema is very old and is
incorrect.

This schema was added to the discribution long ago when it was
not clear what schema to use for Linux autofs information.

The schema was updated (at least to have consistent OIDs) somewhere
along the line but the autofs distribution copy was never updated.
The schema has now been updated in autofs but it is not recommended
for use as the schema for autofs map information.

The rfc2307 or, preferably the, rfc2307bis schema is the recommened
schema to use.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG             |    1 +
 README.autofs-schema  |   18 ++++++++++++++++++
 samples/autofs.schema |   24 ++++++++++--------------
 3 files changed, 29 insertions(+), 14 deletions(-)
 create mode 100644 README.autofs-schema

diff --git a/CHANGELOG b/CHANGELOG
index 601a241..ab44b76 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -21,6 +21,7 @@ xx/xx/2016 autofs-5.1.3
 - fix argc off by one in mount_autofs.c.
 - fix _strncmp() usage.
 - fix create_client() RPC client handling.
+- update and add README for old autofs schema.
 
 15/06/2016 autofs-5.1.2
 =======================
diff --git a/README.autofs-schema b/README.autofs-schema
new file mode 100644
index 0000000..c121e1c
--- /dev/null
+++ b/README.autofs-schema
@@ -0,0 +1,18 @@
+autofs schema
+=============
+
+The distribution file samples/autofs.schema is very old and is
+incorrect.
+
+This schema was added to the discribution long ago when it was
+not clear what schema to use for Linux autofs information.
+
+The schema was corrected somewhere along the line but the autofs
+distribution copy was never updated. The schema has now been
+updated but it is not recommended for use as the schema for autofs
+map information.
+
+The rfc2307 or, preferably the, rfc2307bis schema is the recommened
+schema to use.
+
+Ian
diff --git a/samples/autofs.schema b/samples/autofs.schema
index 98d78c4..07e23b4 100644
--- a/samples/autofs.schema
+++ b/samples/autofs.schema
@@ -1,6 +1,3 @@
-#
-# $id$
-#
 # Depends upon core.schema and cosine.schema
 
 # OID Base is 1.3.6.1.4.1.2312.4
@@ -11,17 +8,16 @@
 
 # Attribute Type Definitions
 
-attributetype ( 1.3.6.1.1.1.1.25 NAME 'automountInformation'
-        DESC 'Information used by the autofs automounter'
-        EQUALITY caseExactIA5Match
-        SYNTAX 1.3.6.1.4.1.1466.115.121.1.26 SINGLE-VALUE )
+attributetype ( 1.3.6.1.4.1.2312.4.1.2 NAME 'automountInformation'
+	DESC 'Information used by the autofs automounter'
+	EQUALITY caseExactMatch
+	SYNTAX 1.3.6.1.4.1.1466.115.121.1.26 SINGLE-VALUE )
 
-objectclass ( 1.3.6.1.1.1.1.13 NAME 'automount' SUP top STRUCTURAL
-        DESC 'An entry in an automounter map'
-        MUST ( cn $ automountInformation $ objectclass )
-        MAY ( description ) )
+objectclass ( 1.3.6.1.4.1.2312.4.2.3 NAME 'automount' SUP top STRUCTURAL
+	DESC 'An entry in an automounter map'
+	MUST ( cn $ automountInformation $ objectclass )
+	MAY ( description ) )
 
 objectclass ( 1.3.6.1.4.1.2312.4.2.2 NAME 'automountMap' SUP top STRUCTURAL
-        DESC 'An group of related automount objects'
-        MUST ( ou ) )
-
+	DESC 'An group of related automount objects'
+	MUST ( ou ) )
